docs/specification/appendix.md

```diff
@@ -118,4 +124,67 @@ The `ros2` profile describes how to create MCAP files for [ROS 2](https://docs.r
 
 #### Schema
 
-- `encoding`: MUST contain `ros2msg`
+- `encoding`: MUST contain either `ros2msg` or `ros2idl`
+
+## `gendeps`-style message definition delimiting
```

Comment:
I think this documentation should move to the respective `ros2msg` and `ros2idl` schema encoding sections. The profiles should just reference them by name.